utilities/src/main/java/cz/xtf/docker/DockerContainer.java

```diff
@@ -63,9 +61,13 @@ public static DockerContainer createForPod(Pod pod) {
 		return createForPod(pod, pod.getMetadata().getLabels().get("name"));
 	}
 
+
 	public static DockerContainer createForPod(Pod pod, String containerLabel) {
 		String host = pod.getSpec().getNodeName();
+		return createForPod(pod, containerLabel, host);
+	}
 
+	public static DockerContainer createForPod(Pod pod, String containerLabel, String host) {
 		try {
```

Comment:
this part (the whole try-catch block with guessing the hostname from nodeName should be moved to the createForPod(Pod pod, String containerLabel)  

if explicit host is provided, it should just be used as a host, with no additional hacks.

Reply:
Done.